docs/dev/index.rst

```diff
@@ -0,0 +1,139 @@
+Developers Guide
+================
+
+The project is hosted on `GitHub <https://github.com/MagicStack/uvloop>`_.
+and uses `Travis <https://travis-ci.org/MagicStack/uvloop>`_ for
+Continuous Integration.
+
+If you have found a bug or have an idea for an enhancement that would
+improve the library, use the `bug tracker <https://github.com/MagicStack/uvloop/issues>`_.
+
+
+Get the source
+--------------
+
+.. code-block:: console
+
+    $ git clone --recursive git@github.com:MagicStack/uvloop.git
+
+The ``--recursive`` argument is important. It will fetch the ``libuv`` source
+from the `libuv` Github repository.
+
+
+Build
+-----
+
+To build `uvloop`, you'll need `Cython` and Python 3.5.
+
+.. note::
+
+    The best way to work on `uvloop` is to create a virtual env, so that
+    you'll have Cython and Python commands pointing to the correct
+    tools.
+
+    .. code-block:: console
+
+        $ python3 -m venv myvenv
+        $ cd myvenv
+        $ source bin/activate
+        $ cd ..
+
+To build `uvloop` run the ``make`` rule from the top level directory.
+
+.. code-block:: console
+
+    $ cd uvloop
+    $ make
+
+
+Test
+----
+
+The easiest method to run all of the unit tests is to run the ``make test``
+rule from the top level directory. This runs the standard
+library ``unittest`` tool which discovers all the unit tests and runs them.
+It actually runs them twice, once with the `PYTHONASYNCIODEBUG` enabled and
+once without.
+
+.. code-block:: console
+
+    $ cd uvloop
+    $ make test
+
+Alternatively, you can run the tests from the ``tests`` directory using the
+standard library `unittest` approach. When using this approach you must
+specify the ``PYTHONPATH`` so that the `uvloop` package can be resolved from
+down in the ``tests`` directory.
+
+.. code-block:: console
+
+    $ cd uvloop/tests
+    $ PYTHONPATH=$PWD/.. python -m unittest
+
+
+Individual Tests
```

Comment:
I'd recommend to use `py.test` for individual tests -- that's what I'm myself using.  For example:

`$ PYTHONPATH=. py.test -k test_signals_sigint_uvcode`